stripe/src/test/java/com/stripe/android/IssuingCardPinServiceTest.java

```diff
@@ -0,0 +1,151 @@
+package com.stripe.android;
+
+import com.stripe.android.testharness.TestEphemeralKeyProvider;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.MockitoAnnotations;
+import org.robolectric.RobolectricTestRunner;
+
+import java.util.Map;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+/**
+ * Test class for {@link IssuingCardPinService}.
+ */
+@RunWith(RobolectricTestRunner.class)
+public class IssuingCardPinServiceTest {
+
+    private static final String EPHEMERAL_KEY = "{\n" +
+            "  \"id\": \"ephkey_123\",\n" +
+            "  \"object\": \"ephemeral_key\",\n" +
+            "  \"secret\": \"ek_test_123\",\n" +
+            "  \"created\": 1501179335,\n" +
+            "  \"livemode\": false,\n" +
+            "  \"expires\": 1501199335,\n" +
+            "  \"associated_objects\": [{\n" +
+            "            \"type\": \"issuing.card\",\n" +
+            "            \"id\": \"ic_abcd\"\n" +
+            "            }]\n" +
+            "}";
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
we don't need this method if we aren't using mockito annotations

Reply:
oh yeah